src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
```

Comment:
it's better to use `box-sizing: border-box` and set the final sizes (200px width)